scout/server/blueprints/variant/controllers.py

```diff
@@ -644,36 +644,32 @@ def build_clinvar_submission(store, request, institute_id, case_name, variant_id
     updated_submission = store.add_to_submission(open_submission["_id"], submission_objects)
 
 
-def clinvar_export(store, institute_id, case_name, variant_id):
+def clinvar_export(store, institute_obj, case_obj, variants):
     """Gather the required data for creating the clinvar submission form
 
     Args:
         store(scout.adapter.MongoAdapter)
-        institute_id(str): Institute ID
-        case_name(str): case ID
-        variant_id(str): variant._id
+        institute_obj(str): Institute ID
+        case_obj(str): case ID
+        variants(list): list of variant._id strings
```

Comment:
Are these variables strings or instances of a class? I find the code hard to read. In previous version they were named for example `case_name` and naturally interpreted  as a string. Now named `case_obj` but is it a string as the docstring says or an instance of a class, as the name indicates?

Reply:
Fixed